cpu/cortexm_common/include/cpu.h

```diff
@@ -25,13 +25,12 @@
  * @author      Joakim Gebart <joakim.gebart@eistec.se>
  */
 
-#ifndef CORTEXM_COMMON_H_
-#define CORTEXM_COMMON_H_
-
+#ifndef CPU_H_
+#define CPU_H_
 
 #include "cpu_conf.h"
 
-/**
+/*
  * TODO: remove once core was adjusted
```

Comment:
This TODO reference is something that I have been curious about for some time, what does it mean? Is there an open issue for this? Is this related to the COREIF_NG definition found in most (all?) platform makefiles?